src/CronTaskLog.php

```diff
@@ -38,8 +38,13 @@
 /**
  * CronTaskLog class
  **/
-class CronTaskLog extends CommonDBTM
+class CronTaskLog extends CommonDBChild
 {
+    public static $itemtype  = 'CronTask';
+    public static $items_id  = 'crontasks_id';
+
+    public static $rightname        = 'config';
```

Comment:
Is this still necessary ? I guess it will now use the parent rightname.

Reply:
Seems not